hack/pipeline/go.mod

```diff
@@ -0,0 +1,7 @@
+module github.com/projectcalico/calico/hack/pipeline
```

Comment:
I don't think we should be using a separate go.mod / go.sum - both of these dependencies are already in our root go module.